proposals/0001-content-steering.md

```diff
@@ -0,0 +1,98 @@
+- Feature Name: Content Steering
+- Start Date: 2023-07-31
+- PR: https://github.com/videojs/rfcs/pull/1
+
+# Summary
+[summary]: #summary
+
+Content distributors often use multiple Content Delivery Networks (CDNs) to distribute their content to the end-users. They may upload a copy of their catalog to each CDN, or more commonly have all CDNs pull the content from a common origin. Alternate URLs are generated, one for each CDN, that point at identical content. Streaming players may then access alternate URLs in the event of delivery problems.
+
+Historically, alternate URLs have been features of both HLS and DASH for many years, but they only provide means of failover controlled by the player, and not by the content distributor.
+
+Content steering describes a deterministic capability for a content distributor to switch the content source that a player uses either at start-up or midstream, by means of a remote steering service.
+
+# Motivation
+[motivation]: #motivation
+
+Content steering is a relatively new mechanism to allow content distributors greater control over where a player or end-user accesses their HLS or DASH media from, which can be leveraged to provide a better quality of experience. This feature has been implemented by other leading players in the industry and should also be adapted for video.js.
```

Comment:
Question, should it be, `should also be adopted by video.js.` instead of 
> should also be adapted for video.js.

Reply:
Good catch, @amtins! I appreciate you reviewing this.